tiledb/bioimg/helpers.py

```diff
@@ -260,3 +262,50 @@ def is_win_path(scheme: str) -> bool:
 
 def is_local_path(scheme: str) -> bool:
     return True if is_win_path(scheme) or scheme == "" else False
+
+
+def get_logger(level: int = logging.INFO, name: str = __name__) -> logging.Logger:
+    """
+    Get a logger with a custom formatter and set the logging level.
+
+    :param level: logging level, defaults to logging.INFO
+    :param name: logger name, defaults to __name__
+    :return: Logger object
+    """
+
+    sh = logging.StreamHandler(stream=sys.stdout)
+    formatter = logging.Formatter(
+        "[%(asctime)s] [%(module)s] [%(funcName)s] [%(levelname)s] %(message)s"
+    )
+    sh.setFormatter(formatter)
+
+    logger = logging.getLogger(name)
+    # Only add one handler, in case get_logger is called multiple times
+    if not logger.handlers:
+        logger.addHandler(sh)
+        logger.setLevel(level)
+
+    return logger
+
+
+def get_logger_wrapper(
+    verbose: bool = False,
+) -> logging.Logger:
+    """
+    Get a logger instance and log version information.
+
+    :param verbose: verbose logging, defaults to False
+    :return: logger instance
+    """
+
+    level = logging.DEBUG if verbose else logging.INFO
```

Comment:
Am I following correctly that this will _always_ be set to `logging.INFO`? I think we need to be able to turn it off entirely (by default)...

Reply:
The default is indeed `logging.info` level. Even if the info logs are minimal I am totally positive to modify the default logger lever to `warning` so it turns off entirely. What do you think?

Reply:
I would lean toward disabling by default unless you feel strongly otherwise - will defer to you.